pkg/app/http/response/response.go

```diff
@@ -2,99 +2,59 @@ package response
 
 import (
 	"encoding/json"
-	"fmt"
 	"net/http"
-	"time"
-
-	"go.uber.org/zap"
 
 	"github.com/CyberAgentHack/server-performance-tuning-2023/pkg/errcode"
 )
 
 const (
-	applicationJSON = "application/json"
+	keyCacheControl = "cache-control"
+	keyContentType  = "content-type"
+	keyVary         = "vary"
```

Comment:
めっちゃnitsです

canonical formで書いた方が良いかもですかね？
（Goだと裏で変換されるようなのでスタンダードを書くいう意味あいで）
https://ron-liu.medium.com/what-canonical-http-header-mean-in-golang-2e97f854316d

```suggestion
	keyCacheControl = "Cache-Control"
	keyContentType  = "Content-Type"
	keyVary         = "Vary"
```

Reply:
:tashikanine:

Reply:
d98eb03c7bc55457ce0653cbb5d8c261cdaa25e5